aerogear/mobile-security-service
# feat(ui): Disable app from UI (AEROGEAR-8698)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-8698

## What

Allows disabling of all app versions from the UI, with an option of setting a custom disabled message.

## Why

Requirement to meet acceptance criteria.

## Verification Steps
 
1. Go to the app detailed view of an app that has one or more versions.
2. For testing purposes, ensure that at least one of the versions is enabled.
3. Click `Disable App`. Optionally enter a custom message.
4. Click `Save`.
5. Has the UI updated to show your new message and all versions disabled?
6. Refresh the page, your changes should have persisted.
7. If you do not enter a custom disabled message, it should preserve the existing messages attached to each version.

## Checklist:


## Progress
